### PR TITLE
Log missing lines

### PR DESCRIPTION
**Pull request by iota97**
This will log all the missing translation lines on shutdown, in this way it won't clutter the log at runtime and should help us remembering to add them when appropriate.

It also helps spotting typo and paste error like `Color Tint` being in the network section :)

I changed the `I18NCategory::Missed()` to return by copy as I don't think the lock was working before, was it?

### CODE REVIEW

File: assets/lang/zh_CN.ini [hunk 1]
```diff
@@ -1183,5 +1197,6 @@ Download = 下载
 New version of PPSSPP available = 新版本 PPSSPP 可用
 
 [Search]
-Find settings = 搜索设置
-Filter = 筛选
+Filtering settings by '%1' = Filtering settings by '%1'
+#Find settings = 搜索设置
+#Filter = 筛选
```
Comment (unknownbrackets) [inline assets/lang/zh_CN.ini:1202]: Hm, these are used as long as your device has a keyboard.

-[Unknown]

File: assets/lang/zh_CN.ini [hunk 2]
```diff
@@ -627,7 +628,7 @@ Graphics = 图像设置
 Networking = 网络设置
 System = 系统设置
 Tools = 工具
-Search = 搜索
+#Search = 搜索
```
Comment (unknownbrackets) [inline assets/lang/zh_CN.ini:631]: This should be used if the device has a keyboard, right?

-[Unknown]

**Review by unknownbrackets — COMMENTED** [review]
I think this is reasonable, although if you're on Windows (where the log console is a separate window, even if you run from command line), it won't help.  That doesn't make it not useful, though.

-[Unknown]

File: assets/lang/zh_CN.ini (hunk 1 above)
Comment (iota97) [reply assets/lang/zh_CN.ini:1202]: Ufff, nice catch.

I run the lang tools to sync missing line. Seems like it commended unused one too for some reason?

SDL build doesn't have the search function, anyway I'll have to fix this.

**Pull request merged**